docs/superpowers/specs/2026-04-30-issue-135-bench-baseline.md

```
@@ -0,0 +1,150 @@
+## Results
+
+All times in nanoseconds.
+
+| Fixture | Mean | Std Dev | 95% CI lower | 95% CI upper | Median |
+|---|---|---|---|---|---|
+| `lsp_diagnostics/diagnostics/small_10` | 1,402,478 (≈1.40 ms) | 5,653 | 1,399,995 | 1,404,848 | 1,403,700 |
+| `lsp_diagnostics/diagnostics/medium_50` | 11,217,056 (≈11.22 ms) | 52,528 | 11,194,021 | 11,238,737 | 11,218,752 |
+
+## Raw Criterion summary
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)